### DISCUSSION TITLE
Login/register with existing platforms

### DISCUSSION THREAD

**Discussion by jankapunkt** [openqda/openqda#134] [Polls]
The OAuth2 protocol enables to authorize (but also authenticate) users by a trusted third-party. This increases likelyhood and ease of use and prevents the need for another password.

**Comment by kadewe** [comment]
Google is often used, but maybe shouldn't for scientific use?

**Comment by kadewe** [comment]
Shibboleth is often used

**Comment by kadewe** [comment]
What is the IDP service at https://idp.uni-bremen.de?

**Reply by jankapunkt** [reply]
This is the ZfN identity provier, likely linked to ldap, shibboleth, OAuth2 etc.

**Comment by kadewe** [comment]
Apple users could use "Register with Apple". I think basically a lot of the students will opt in to use Google or Apple auths, but from a science point of view this is not optimal. But we need to ask our users.

**Reply by jankapunkt** [reply]
I think both user groups are important which is why would currently vote for

- ORCID
- DFN/Shibboleth
- Apple
- Google

**Reply by jankapunkt** [reply]
This is a list of providers that are supported by the laravel package https://socialiteproviders.com/Microsoft/#installation-basic-usage

**Reply by kadewe** [reply]
oh my god, much too many. I like ORCID, DFN/Shibboleth, Apple and Google.
